mobile/src/main/java/rs/readahead/washington/mobile/views/fragment/googledrive/viewpagerfragments/DraftsGoogleDriveFragment.kt

```diff
@@ -0,0 +1,65 @@
+package rs.readahead.washington.mobile.views.fragment.googledrive.viewpagerfragments
+
+import android.annotation.SuppressLint
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.lifecycleScope
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import rs.readahead.washington.mobile.R
+import rs.readahead.washington.mobile.domain.entity.reports.ReportInstance
+import rs.readahead.washington.mobile.views.fragment.googledrive.GoogleDriveViewModel
+import rs.readahead.washington.mobile.views.fragment.main_connexions.base.BUNDLE_REPORT_FORM_INSTANCE
+import rs.readahead.washington.mobile.views.fragment.main_connexions.base.BaseReportsFragment
+import rs.readahead.washington.mobile.views.fragment.main_connexions.base.BaseReportsViewModel
+import rs.readahead.washington.mobile.views.fragment.main_connexions.base.ReportsUtils
+
+@AndroidEntryPoint
+class DraftsGoogleDriveFragment : BaseReportsFragment() {
+
+    private val viewModel by viewModels<GoogleDriveViewModel>()
+
```

Comment:
Maybe we should unify the implementation of the `ViewModel` as use it all over as an attribute like this